src/import/sst_service.rs

```diff
@@ -62,34 +62,42 @@ impl ImportSst for ImportSSTService {
         let timer = Instant::now_coarse();
 
         let token = self.importer.token();
+        let thread1 = self.threads.clone();
+        let thread2 = self.threads.clone();
         let import1 = self.importer.clone();
         let import2 = self.importer.clone();
-        let bounded_stream = mpsc::spawn(stream, &self.pool, self.cfg.stream_channel_size);
+        let bounded_stream = mpsc::spawn(stream, &self.threads, self.cfg.stream_channel_size);
 
         ctx.spawn(
             bounded_stream
                 .map_err(Error::from)
                 .for_each(move |chunk| {
-                    let start = Instant::now_coarse();
-                    if chunk.has_meta() {
-                        import1.create(token, chunk.get_meta())?;
-                    }
-                    if !chunk.get_data().is_empty() {
-                        let data = chunk.get_data();
-                        import1.append(token, data)?;
-                        IMPORT_UPLOAD_CHUNK_SIZE.observe(data.len() as f64);
-                    }
-                    IMPORT_UPLOAD_CHUNK_DURATION.observe(start.elapsed_secs());
-                    Ok(())
+                    let import = import1.clone();
```

Comment:
How about `let import = import1;`? Or just remove this line.

Reply:
This line is actually necessary, because we are in a stream and `import1` will be moved multiple times.